lib/linux.js

```diff
@@ -148,9 +148,20 @@ async function setAvailableApps() {
 	}
 }
 
+// Get the current active desktop session
+async function getCurrentDesktopSession() {
+    try {
+        const {stdout} = await exec('echo $DESKTOP_SESSION');
```

Comment:
Why not just `process.env.DESKTOP_SESSION`?

Reply:
ok I will try it, Thanks